sdr/concepts_and_interactions.md

```diff
@@ -0,0 +1,202 @@
+  * preservation-client is a ruby gem for interating with pres cat's REST API
+* Once preservation_catalog has been made aware that new object has been preserved or a new version has been preserved for an existing object, assuming the object passes structural and checksum validation, preservation catalog will replicate the new object version to the cloud.
+  * It creates a zip file (uncompressed) of the new Moab version, and sends that to each of the S3 endpoints (there are currently 3, 2 AWS and 1 IBM).
+* How is redis used in preservation
+  * there is a Redis instance that coordinates all of the regular accessioning "robots" (including preservation_robots)
+  * there is a Redis instance that coordinates all of the preservation_catalog workers (for replicating copies to the cloud and for auditing all preserved content both on prem in in the cloud)
+* The preservation catalog codebase contains a database README with an ER diagram for the schema as well as a number of useful example queries, and a diagram and README for its replication worker pipeline.
+
+## Technical Metadata app
+_Service to extract and record technical metadata for files deposited into SDR_
+
+(ask JLitt)
+
+* technical metadata: for each file, its file type, characteristics of image, PDF, A/V files and so on
+
+## SDR Tags database
+_Accessioneers and Argo users and some of our apps "tag" digital objects in the SDR_
+
+(ask Mike)
+
+  * A table in dor-services-app's PostgreSQL database.  Used to live in Fedora.  Useful for human administration of repository content in Argo.
+
+## Modsulator
+_A gem and an app that can turn spreadsheets into MODS in a single bound ... or something_
+
+## Bulk jobs/actions in Argo
+_When making changes to objects one at a time is the wrong approach_
+
+## Web archiving
+_yes, Virginia, DLSS does crawl or download to get WARCs and ingest them into SDR and serve them out_
+
+(ask JLitt or Naomi)
+
+  * was-registrar-app
+  * was_robot_suite
+  _robots to carry out special processing required for WAS seed and crawl objects_
+  * SWAP
+  * wasapi-downloader
+  * ...
+
+## Solr indexing of SDR
+_How we currently index SDR content for searching; the index is mostly used by Argo._
+
+(ask JCoyne)
+
+* dor_indexing_app - receives a REST call when an object changes
+* Karaf, Camel, ... - the REST call to dor_indexing_app is currently queued in our ActiveMQ instance e.g. when Fedora emits a message indicating that an object was modified.
+
+## Events (e.g. in Argo object view)
+_A way to capture an objects changes over time_
+
+(ask JCoyne)
+
+* A table in dor-services-app's PostgreSQL database.  Other applications make REST calls to DSA (token authenticated), logging notable activity that might help future with future debugging or provenance determination.
+
+## Goobi
+_A workflow tool used by DLSS digitization staff and then hooking into common-accessioning_
+
+(ask Peter)
+
+- what is it, and how does it relate to SDR
+
+## sul_pub
```

Comment:
SulPub/DLME/Sinopia are not SDR concepts and probably don't belong here.

Reply:
could have a separate page for each, or one for all the non SDR services AFAIC

Reply:
moved to the listing for those apps in the main index, for now, since what info was there was short.  can break out later.